### PR TITLE
Responsive Footer Created

### PR DESCRIPTION
**Pull request by vedant-c**
This is a solution for issue #94.
Please check the proposed solution at - https://vedant-c.github.io/skill-board/login.html
I have created a footer containing the skillboard logo and links to the skillboard GitHub repository as well as the devscollab Instagram handle. The footer is made responsive and works well on large(desktop) as well as small(phone) screen. I have tried to match the theme of the website. Please let me know if there are any changes required regarding the content or the design as well.

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -60,6 +66,27 @@
...
+              <div class="col-sm-6 d-flex my-3 py-auto flex-column align-items-center order-sm-1 order-2">
+                <img src="./assets/images/logo.png" class='navLogo ' alt="..">
+              </div>
+              <div class="col-sm-6 d-flex my-3 py-auto flex-column align-items-center justify-content-center
+                           footer-content order-md-2 order-1">
+                <h4>Check Out</h4>
+                <ul class='list-unstyled'>
+                  <li><a href="https://github.com/devscollab/skill-board" target="_blank"><i class="fab fa-github fa-2x footer-icon"></i></a></li>
+                  <li><a href="https://www.instagram.com/devscollab/?hl=en" target="_blank"><i class="fab fa-instagram fa-2x footer-icon"></i></a></li>
+                </ul>
```
Comment (tejasmorkar) [inline login.html:83]: This list can be made horizontal by using display: flex and flex-direction: row

File: login.html [hunk 2]
```diff
@@ -60,6 +66,27 @@
         <img src="./assets/images/blob.png" id="blob">
     </div>
 
+    <!-- FOOTER -->
+    <div class="footer">
+        <div class="row mx-0">
+          <div class="col-md-6 mx-auto">
+            <div class="row mx-0 align-items-center">
+              <div class="col-sm-6 d-flex my-3 py-auto flex-column align-items-center order-sm-1 order-2">
+                <img src="./assets/images/logo.png" class='navLogo ' alt="..">
```
Comment (tejasmorkar) [inline login.html:75]: Image size can be minimized

**Review by tejasmorkar — CHANGES_REQUESTED** [review]
Hi, thank you for the wonderful contribution to the project. 

I have commented a few changes in the code review in order to make the footer vertical size smaller than it is now.  You can make those changes and push them to your repo.

File: login.html (hunk 1 above)
Comment (vedant-c) [reply login.html:83]: Okay, I will do the changes

File: login.html (hunk 2 above)
Comment (vedant-c) [reply login.html:75]: How small do we need it?

Comment (tejasmorkar) [reply login.html:75]: 56px is fine

Comment (vedant-c) [reply login.html:75]: I tried it and it looks too small. Do you want me to stick with 56?

Comment (vedant-c) [reply login.html:75]: According to me, 70px looks fine. What do you say?

Comment (tejasmorkar) [reply login.html:75]: Sure, anything that you think is fine.

**Comment by vedant-c** [comment]
Now do i have to make a new pull request or just push the changes?

**Comment by vedant-c** [comment]
I have pushed the changes. Please have a look

**Comment by tejasmorkar** [comment]
> Now do i have to make a new pull request or just push the changes?

Just pushing the work updates the PR

**Review by tejasmorkar — APPROVED** [review]

**Pull request merged**